### PR TITLE
Always Green light for nightly

### PR DESCRIPTION
**Pull request by sosiristseng**
Using [`continue-on-error`](https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#jobsjob_idcontinue-on-error)

### CODE REVIEW

**Comment by sosiristseng** [comment]
And I don't know why Codecov Report became scary. (but the coverage slightly increased)

**Comment by stevengogogo** [comment]
> And I don't know why Codecov Report became scary. (but the coverage slightly increased).
> 
> Attempting to access the report yielded Github API rate limit error.

Still have the same coverage though display is weird.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -6,10 +6,17 @@ jobs:
   test:
     name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }} - ${{ github.event_name }}
     runs-on: ${{ matrix.os }}
+    continue-on-error: ${{ matrix.version == 'nightly' }}
     strategy:
       matrix:
-        version: ['1.6', '^1.7.0-rc1', 'nightly']
-        os: [ubuntu-latest, windows-latest, macOS-latest]
+        version:
```
Comment (stevengogogo) [inline .github/workflows/ci.yml:12]: Why this version is recommended?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -6,10 +6,17 @@ jobs:
...
-        version: ['1.6', '^1.7.0-rc1', 'nightly']
-        os: [ubuntu-latest, windows-latest, macOS-latest]
+        version: 
+          - '1.6'
+          - '^1.7.0-0'
+          - 'nightly'
+        os: 
+          - ubuntu-latest
+          - windows-latest
+          - macOS-latest
```
Comment (stevengogogo) [inline .github/workflows/ci.yml:19]: It looks like macOS VM is problematic. Should we keep it?

**Review by stevengogogo — COMMENTED** [review]
I guess this change works same as the original one, but just in different format?

File: .github/workflows/ci.yml (hunk 2 above)
Comment (sosiristseng) [reply .github/workflows/ci.yml:19]: Keep it, why not? I suspect it's the cache problem and we can fix that later.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (sosiristseng) [reply .github/workflows/ci.yml:12]: This form clutters less if you have a long list of versions later on. The two forms are basically the same.

**Comment by sosiristseng** [comment]
Yeah I just made the tests always pass on Julia nightly. Nothing else is changed.

**Comment by stevengogogo** [comment]
> Yeah I just made the tests always pass on Julia nightly. Nothing else is changed.

Any progress. It looks MacOS is still errornous

**Comment by stevengogogo** [comment]
Ready to fly

**Pull request merged**